activemq/metrics.yaml

```diff
@@ -0,0 +1,55 @@
+# List of metrics to be collected by the integration
+# Read http://docs.datadoghq.com/integrations/java/ to learn how to customize it
```

Comment:
I think we shouldn't encourage customising this file.
I'd put a "don't touch this, use conf.yaml instead" kind of advice